src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,25 @@
+package seedu.addressbook.data.person;
+
+public class Contact {
+    
+    public String value;
+    private boolean isPrivate;
+    
+    public Contact(String contact, boolean isPrivate){
+	this.isPrivate = isPrivate;
```

Comment:
It is strange that you created a constructor that requires both "contact" and "isPrivate" arguments but did not use the "contact" argument at all.